### DISCUSSION TITLE
How to renew a certificate when an old certificate is already expired

### DISCUSSION THREAD

**Discussion by TaKO8Ki** [phax/phoss-smp#283] [Q&A] [answered]
Hi. Thank you for maintaining phoss-smp. 

I misunderstood the expiration date of the SMP certificate (July 22) in the test environment and obtained a new certificate one day after it expired. Subsequently, I attempted to update the certificate but received a warning on the following page that the SML certificate update must be performed before the old certificate expires to update to the new certificate. Is there any solution to this situation? I would appreciate any advice.

Do I need to unregister our SMP and deleting all participants / Service Groups once and re-register it?

<img width="672" alt="スクリーンショット 2024-07-24 17 11 11" src="https://github.com/user-attachments/assets/65a9728d-99a7-4c26-82ab-9337f0a0830f">

**Comment by phax** [comment, marked as answer]
@TaKO8Ki no worries, that happens to the best :)

Please see https://peppol.helger.com/public/locale-en_US/menuitem-docs-peppol-cert-update#smp for an explanation on what and how to do in this case.

hth

**Reply by TaKO8Ki** [reply]
Ok. Thanks. I will contact [EC eDelivery support](https://peppol.helger.com/public/menuitem-docs-sml-support)!

**Reply by TaKO8Ki** [reply]
I have sent the email :)
